vizapp/viewer/simple_bqplot_image.py

```diff
@@ -0,0 +1,73 @@
+# This provides a simplified version of the bqplot image viewer
+
+from ipywidgets import Button, VBox, HBox
+
+from glue.core.application_base import Application
+from glue_jupyter.bqplot.image import BqplotImageView
+from glue_jupyter.widgets import LinkedDropdown
+from glue_jupyter.common.slice_helpers import MultiSliceWidgetHelper
+
+__all__ = ['SimpleBqplotImageViewer', 'simple_imshow']
+
+
+def simple_imshow(app, data=None):
+    viewer = Application.new_data_viewer(app, SimpleBqplotImageViewer)
```

Comment:
By calling ``Application.new_data_viewer`` rather than ``app.new_data_viewer`` we bypass the showing of the widgets by default. This approach can be used with other viewers too.